examples/image-streamer/build_plan.rb

```diff
@@ -21,12 +28,27 @@
   name: 'Build_Plan_1',
   oeBuildPlanType: 'Deploy'
 }
-
-plan_script = OneviewSDK::ImageStreamer::API300::PlanScript.find_by(@client, name: @plan_script1_name).first
-plan_script2 = OneviewSDK::ImageStreamer::API300::PlanScript.find_by(@client, name: @plan_script2_name).first
+options_plan_script = {
+  description: 'Description of this plan script',
+  name: @plan_script2_name,
+  hpProvided: false,
+  planType: 'deploy',
+  content: 'esxcli system hostname set --domain "@DomainName@"'
+}
+plan_script_class = OneviewSDK::ImageStreamer.resource_named('PlanScript', @client.api_version)
+plan_script = plan_script_class.find_by(@client, name: @plan_script1_name).first
```

Comment:
Is this variable `plan_script` used anywhere?

Reply:
at line 57

Reply:
Ok.